vernac/loadpath.ml

```diff
@@ -176,8 +176,9 @@ let locate_file fname =
 (************************************************************************)
 (*s Locate absolute or partially qualified library names in the path *)
 
-type locate_error = LibUnmappedDir | LibNotFound
-type 'a locate_result = ('a, locate_error) result
+module Error = struct
+  type t = LibUnmappedDir | LibNotFound
+end
```

Comment:
maybe we don't need this change

Reply:
Maybe, on the other hand it's done and I find removing these kind of useless alias good.

Reply:
I mean the wrapping in a module of the error type

Reply:
Putting the constructors in modules indeed was indeed not needed, but a practice that is much preferred this days as to avoid having to add prefixes to everything.